forms-flow-api-utils/requirements.txt

```diff
@@ -1,47 +1,19 @@
-Flask-Caching==2.0.1
-Flask-Migrate==3.1.0
-Flask-Moment==1.0.4
-Flask-SQLAlchemy==2.5.1
-Flask==2.1.3
-Jinja2==3.1.2
-Mako==1.2.1
-MarkupSafe==2.1.1
-PyJWT==2.4.0
-SQLAlchemy-Utils==0.38.3
-SQLAlchemy==1.4.40
-Werkzeug==2.1.2
-alembic==1.8.1
-aniso8601==9.0.1
-attrs==22.1.0
-cachelib==0.9.0
-certifi==2022.6.15
-charset-normalizer==2.1.0
-click==8.1.3
-ecdsa==0.18.0
-flask-jwt-oidc==0.3.0
-flask-marshmallow==0.14.0
-flask-restx==0.5.1
-gunicorn==20.1.0
-idna==3.3
-importlib-metadata==4.12.0
-itsdangerous==2.1.2
-jsonschema==4.9.1
-marshmallow-sqlalchemy==0.28.1
-marshmallow==3.17.0
-packaging==21.3
-psycopg2-binary==2.9.3
-pyasn1==0.4.8
-pyparsing==3.0.9
-pyrsistent==0.18.1
-python-dotenv==0.20.0
-python-jose==3.3.0
-pytz==2022.2.1
-requests==2.28.1
-rsa==4.9
-six==1.16.0
-urllib3==1.26.11
-PyJWT==2.4.0
-selenium==3.141.0
-selenium-wire==3.0.0
-ndg-httpsclient
-pyopenssl
\ No newline at end of file
+gunicorn
```

Comment:
Since this is python module, sticking versions would cause issues with downstream systems which uses this module.

Reply:
👍